src/utils/stats-data.ts

```diff
@@ -10,8 +10,17 @@ import {
 import { formatContractResult } from './LockOverviewUtils'
 
 const everipediaBaseApiEndpoint = 'https'
-// TODO: get apis for hardcoded values
-const eosVolume = 10121454884
+const getEosSupply = async () => {
+  try {
+    const response = await fetch(
+      'https://www.api.bloks.io/tokens/IQ-eos-everipediaiq',
+    )
+    const result = await response.json()
+    return result[0].supply.circulating
+  } catch (err) {
+    return 10121454884
```

Comment:
better show 0 and log the error. Otherwise we wont know if there is a problem or not